system-addon/content-src/components/Search/_Search.scss

```diff
@@ -22,9 +21,11 @@
   width: 100%;
 
   input {
-    border: 0;
+    background: var(--newtab-searchbox-background-color);
+    border: solid 1px var(--newtab-searchbox-border-color);
     border-radius: $search-border-radius;
     box-shadow: $shadow-secondary, 0 0 0 1px $black-15;
```

Comment:
There's various blacks here that probably need to be themed. E.g., search button hover, etc.